storybook/stubs/shared/stores/send/TransactionStore.qml

```diff
@@ -189,6 +189,10 @@ QtObject {
     function setSelectedTokenName(tokenName) {
     }
 
+    function setRouteDisabledChains(chainId, disabled) {
+
+    }
+
```

Comment:
Yeah the problem is though that without this stub store, the SendModal would be completely broken in StoryBook; there's a lot of code snippets that rely on the store methods to be there; without them, the JS code flow just breaks

Reply:
And as for the proposed change: this is just a bugfix PR (for 2.30 eventually) and I'd definitely postpone any refactorings for the start of the next milestone

Reply:
This is no risk at all, just cut from the stub and paste in `SendModalPage`:

![image](https://github.com/user-attachments/assets/36aba8e3-662c-4992-9b30-bcf002f668e8)

`0` impact on the app, `0` risk. But instead of adding piece of code violating agreements, it will be left in a bit better shape, and will prevent from further violations of that kind there :)

Reply:
Not that simple, I tried; it needs dozens of all those methods like `setRouteDisabledChains()` I just added above

Reply:
I don't know why I didn't move it instead of writing this disclaimer... :upside_down_face:

Reply:
> Not that simple, I tried; it needs dozens of all those methods like `setRouteDisabledChains()` I just added above

Whatever you added, just move it from the stub to the `SendModalPage`. The stub is used only in send modal so all the current content is needed and can be just moved.

Reply:
Done

Reply:
> Ah got it now, will do; I thought you wanted me to remove the whole stub TransactionStore right now :)

Omg, it would be something :rofl:

Reply:
I actually meant to move the function you added along with the rest of stuff and removing disclaimer and leaving only plaing `QtObject {}` there. Moving a lot but just cut/paste basically.

Reply:
I'm approving as it's kind of "optional" work here anyways...